linera-rpc/proto/rpc.proto

```diff
@@ -120,6 +120,12 @@ message VersionInfo {
     string wit_hash = 6;
 }
 
+message NetworkDescription {
+    string name = 1;
+    CryptoHash genesis_config_hash = 2;
+    uint64 timestamp = 3;
+}
```

Comment:
`name` and `genesis_config_hash` are self-explanatory but `timestamp` is not IMHO. Is it a timestamp at the time of generating the instance of `NetworkDescription`? Is it a timestamp of creating the message? Is it a timestamp of generating the genesis config? 

Similarly for the `NetworkDescription` struct.